aspnet/fundamentals/configuration.rst

```diff
@@ -41,7 +40,6 @@ It's not unusual to store configuration values in a hierarchical structure, espe
 .. _config-json:
 
 .. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/appsettings.json
-  :linenos:
   :language: json
 
 The application uses configuration to configure the right connection string. Access to the ``ConnectionString`` setting is achieved through this key: ``Data:DefaultConnection:ConnectionString``.
```

Comment:
We should update the appsettings.json to use the new RC2 pattern:

```json
  "ConnectionStrings": {
    "DefaultConnection": "<connection string here>"
  }
```

And then to access it you can use the new `config.GetConnectionString("DefaultConnection")`.